include/mega/megaclient.h

```diff
@@ -1159,16 +1165,24 @@ class MEGA_API MegaClient
     pendinghttp_map pendinghttp;
 
     // record type indicator for sctable
-    enum { CACHEDSCSN, CACHEDNODE, CACHEDUSER, CACHEDLOCALNODE, CACHEDPCR, CACHEDTRANSFER, CACHEDFILE, CACHEDCHAT, CACHEDSTATUS } sctablerectype;
+    enum { CACHEDSCSN, CACHEDNODE, CACHEDUSER, CACHEDLOCALNODE, CACHEDPCR, CACHEDTRANSFER, CACHEDFILE, CACHEDCHAT} sctablerectype;
+
+    // record type indicator for statusTable
+    enum { CACHEDSTATUS } statusTableRecType;
```

Comment:
Unused declaration of this enum. You just want to declare the enumeration, not a variable, doesn't you?
```suggestion
    enum StatusTableRecType { CACHEDSTATUS };
```

Reply:
Done in my last commit